src/components/scenes/FioAddressDetailsScene.js

```diff
@@ -172,7 +171,7 @@ const getStyles = cacheStyles((theme: Theme) => ({
 }))
 
 export const FioAddressDetailsScene = connect((state: RootState) => {
-  const fioWallets = getFioWallets(state)
+  const fioWallets = state.ui.wallets.fioWallets
```

Comment:
could we make in this way?
```
const { fioWallets } = state.ui.wallets
```

or maybe even this: 
```
const out: StateProps = {
    fioWallets: state.ui.wallets.fioWallets
  }
  return out
```

Reply:
Good catch, I will put a return type of the `mapStateToProps` method, and use destructuring.